apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/AssemblyInfo.cs

```diff
@@ -29,4 +29,12 @@
     + "a71bc02b7ebf586dbe1f2493c0ab20423ababfd15ce74d2264a6b37745f3658f016abaad662182"
     + "aaef634a60f1346fcc45343acab5b6781535a3134818e13fac895a6c106c0480e34bbb06cb123e"
     + "5583d8d2"
+)]
+
+[assembly: InternalsVisibleTo("Google.Cloud.Spanner.Data.Tests, PublicKey="
```

Comment:
What internals do we have in this assembly that we need access to for these tests?